### PR TITLE
Add support for statistics on Python 3.

### PR DESCRIPTION
**Pull request by jaraco**
❓ **What kind of change does this PR introduce?**
  - [X] 🐞 bug fix
  - [ ] 🐣 feature
  - [ ] 📋 docs update
  - [ ] 📋 tests/coverage improvement
  - [ ] 📋 refactoring
  - [ ] 💥 other

📋 **What is the related issue number (starting with `#`)**

#99

❓ **What is the current behavior?** (You can also link to an open issue here)

Server crashes when statistics are enabled on Python 3.

❓ **What is the new behavior (if this is a feature change)?**

No crash and statistics are counted. New tests ensure the required interface.

📋 **Other information**:

📋 **Checklist**:

  - [X] I think the code is well written
  - [X] I wrote [good commit messages][1]
  - [ ] I have [squashed related commits together][2] after the changes have been approved
  - [X] Unit tests for the changes exist
  - [X] Integration tests for the changes exist (if applicable)
  - [X] I used the same coding conventions as the rest of the project
  - [X] The new code doesn't generate linter offenses
  - [x] Documentation reflects the changes
  - [X] The PR relates to *only* one subject with a clear title
  and description in grammatically correct, complete sentences

[1]: http://chris.beams.io/posts/git-commit/
[2]: https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit

---
This change is [Reviewable](https://reviewable.io/reviews/cherrypy/cheroot/186)

### CODE REVIEW

File: cheroot/test/test_makefile.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
+"""self-explanatory."""
```
Comment (webknjaz) [inline cheroot/test/test_makefile.py]: not really...

Comment (jaraco) [reply cheroot/test/test_makefile.py]: I could write "Tests for the makefile module.", but I feel that's redundant to the name of this module. What would you recommend? Feel free to enhance it.

Comment (jaraco) [reply cheroot/test/test_makefile.py]: BTW, I wouldn't have bothered writing a docstring at all were it not required by the linters.

Comment (webknjaz) [reply cheroot/test/test_makefile.py]: Yeah, I have a different policy now. I think it's useful to mention what to expect from the tests inside the module without forcing the reader to actually read tests.

Comment (jaraco) [reply cheroot/test/test_makefile.py]: That sounds like a maintenance nightmare... trying to keep documentation about tests up-to-date with the code, especially when the documentation isn't adjacent to the code seems like you're just asking for trouble. As much as possible, I try to make code self-documenting and any documentation should expand on or clarify the purpose of a given piece of code and generally not restate what the code describes. Anyway, that's my approach. You're welcome to pursue more aggressive documentation strategies.

Comment (webknjaz) [reply cheroot/test/test_makefile.py]: I'm thinking more about something answering the WHY question. High-level, mostly not requiring updates over time. What this test suite is for, how it's different from others.

**Pull request merged**